tutorial/lesson_19_wrapper_funcs.cpp

```diff
@@ -0,0 +1,405 @@
+// Halide tutorial lesson 19: Wrapper Funcs
+
+// This lesson demonstrates how to use Func::in and ImageParam::in to
+// schedule a Func differently in different places and to stage loads from
```

Comment:
Oxford comma would clarify that there are two distinct uses: "…places, and to stage…"